Sources/KSCrashHandler.swift

```diff
@@ -6,128 +6,135 @@
-	public var tags: EventTags? {
-		didSet { updateUserInfo()}
+
+	// MARK: - Attributes
+
+	private let installation: KSCrash?
+
+	private let keyUser = "user"
+	private let keyEventTags = "event_tags"
+	private let keyEventExtra = "event_extra"
+	private let keyBreadcrumbsSerialized = "breadcrumbs_serialized"
+
+
+	// MARK: - Initializers
+
+	private init() {
+		self.installation = KSCrash.sharedInstance()
+
+		if installation == nil {
+			SentryLog.Error.log("There is no KSCrash.shareInstance for some unknown reason")
+		}
 	}
-	public var extra: EventExtra? {
+
+
+	// MARK: - EventProperties
+
+	internal var tags: EventTags? {
 		didSet { updateUserInfo() }
 	}
-	public var user: User? {
+	internal var extra: EventExtra? {
 		didSet { updateUserInfo() }
 	}
-	
-	public var breadcrumbsSerialized: BreadcrumbStore.SerializedType? {
+	internal var user: User? {
 		didSet { updateUserInfo() }
 	}
-	
-	/// Creates an SentryKSCrashHandler
-	public override init() {
-		self.installation = KSCrash.sharedInstance()
-		
-		if installation == nil {
-			SentryLog.Error.log("There is no KSCrash.shareInstance for some unknown reason")
-		}
-		
-		super.init()
+
+
+	// MARK: - CrashHandler
+
+	internal var breadcrumbsSerialized: BreadcrumbStore.SerializedType? {
+		didSet { updateUserInfo() }
 	}
-	
-	/// Starts the crash reporting and sends any crash reports that
-	/// were capture in previous runs
-	/// - Parameter createdEvent: A closure that passes in a created event
-	public func startCrashReporting(createdEvent: (event: Event) -> ()) {
-	
-		// Temporarily setting introspect to false
-		// because of KSCrash bug
-		// https://github.com/kstenerud/KSCrash/issues/110
+
+	/*
+	Starts the crash reporting and sends any previously saved crash reports
+	- Parameter createdEvent: A closure that passes in a created event
+	*/
+	internal func startCrashReporting(createdEvent: (generatedEvent: Event) -> ()) {
+
+		// Temporarily sets introspect to false due to KSCrash bug
+		// -> https://github.com/kstenerud/KSCrash/issues/110
 		installation?.introspectMemory = false
-		
-		// Mapping KSCrash reports into an Events
-		installation?.sendAllReportsWithCompletion { (filteredReports, completed, error) -> Void in
+
+		// Maps KSCrash reports in `Events`
+		installation?.sendAllReportsWithCompletion() { (filteredReports, completed, error) -> Void in
 			for report in filteredReports {
-				
 				SentryLog.Debug.log("Found report: \(report)")
-				
+
 				// Make sure report is a dictionary that we can handle
 				guard let report = report as? CrashDictionary else {
 					SentryLog.Error.log("KSCrash report was not of type [String: AnyObject]")
 					return
 				}
-				
-				// Report crash time
+
+				// Extract crash timestamp
 				let timestamp: NSDate = {
 					var date: NSDate?
 					if let timestampStr = report["report"]?["timestamp"] as? String {
-						let dateFormatter = NSDateFormatter()
+						let dateFormatter: NSDateFormatter = NSDateFormatter()
```

Comment:
@joshdholtz habit. It's not needed in Swift but it speeds up compiling to specify types, like declaring `internal` can remove if it bothers you.